test/metabase/api/persist_test.clj

```diff
@@ -43,3 +45,39 @@
         (is (= default-cron
                (get-in (task.persist-refresh/job-info-by-db-id)
                        [(:id db) :schedule])))))))
+
+(deftest persisted-info-by-id-test
+  (with-setup db
+    (t2.with-temp/with-temp
+      [:model/Card          model     {:database_id (u/the-id db), :dataset true}
+       :model/PersistedInfo pmodel    {:database_id (u/the-id db), :card_id (u/the-id model)}]
```

Comment:
```suggestion
      [:model/Card          model {:database_id (u/the-id db), :dataset true}
       :model/PersistedInfo pinfo {:database_id (u/the-id db), :card_id (u/the-id model)}]
```
nit: we often align whitespace like this

Reply:
Thanks, I should have done that 😄 

Committed in https://github.com/metabase/metabase/pull/33776/commits/7490ff4c84852e28c2b5b6a71171a6f45ede1df4